news/other-2979.md

```diff
@@ -0,0 +1 @@
+`scl`: Moved `scl` files to the core package.
```

Comment:
This is a packaging/debian related change, please add this information to the description.

Reply:
(I think we should add a dedicated news file category for packaging.)

Reply:
Good suggestion, let's bring this up at the retro :) Thanks!